humility-core/src/hubris.rs

```diff
@@ -3312,17 +3339,7 @@ impl HubrisArchive {
                 // We want to make sure that this is a 32-bit basetype or a
                 // ptrtype.
                 //
-                if let Some(v) = self.basetypes.get(&m.goff) {
-                    let ptr_size = (self.arch.as_ref().unwrap().get_abi_size()
-                        / 8) as usize;
-
-                    if v.size != ptr_size {
-                        return Err(anyhow!(
-                            "expected {} in struct {} ({}) to \
-                            be {} bytes, found to be {} bytes",
-                            member, structure.name, structure.goff, ptr_size, v.size
-                        ));
-                    }
+                if self.basetypes.get(&m.goff).is_some() {
```

Comment:
Empty block?

Reply:
Fixed, just added a `break`.  There was one after the else chain already, but this is cleaner.